website/static/js/osfNotifications.js

```diff
@@ -0,0 +1,7 @@
+var $ = require('jquery');
+var $osf = require('osfHelpers');
+var bootbox = require('bootbox');
+
+//initialize treebeard
+var ProjectNotifications = require('./project-settings-treebeard.js');
```

Comment:
This is a page module. It should go in `website/static/js/pages/osf-notifications.js`. 

See http://cosdev.readthedocs.org/en/latest/osf/js_modules.html